doc/user/content/overview/materialize-cdc.md

```diff
@@ -0,0 +1,237 @@
+
+### Update JSON array
+
+Field | description
+-------- | -----------
+data  | The array that represents the changed record. Each array is composed of objects that define the `name` and `type` of each field to be included in the changefeed.
+time  | The logical timestamp of the update. Materialize uses this to determine the order of updates and, in combination with the progress message, to determine whether there are duplicate updates to be disgarded.
+diff  | The type of update (`-1` for deletion, `1` for addition or upsert)
+
+#### Example update JSON definition
+
+```[
+  {
+    "type": "array",
+    "items": {
+      "type": "record",
+      "name": "update",
+      "namespace": "com.materialize.cdc",
+      "fields": [
+        {
+          "name": "data",
+          "type": {
+            "type": "record",
+            "name": "data",
+            "fields": [
+              {
+                "name": "COLUMNNAME1",
+                "type": "DATATYPE"
+              },
+              {
+                "name": "COLUMNAME2",
+                "type": "DATATYPE"
+              }
+            ]
+          }
+        },
+        {
+          "name": "time",
+          "type": "long"
+        },
+        {
+          "name": "diff",
+          "type": "long"
+        }
+      ]
+    }
+  },
+```
+
+### Progress JSON array
+
+The progress messages describe the number of updates transmitted at a particular timestamp or between specified times.
+
+Field | description
+-------- | -----------
+lower  | The earliest logical timestamp for a set of updates
+upper  | The latest logical timestamp for a set of updates
+counts | The number of updates transmitted between `lower` and `upper`.
+
+#### Example progress message JSON
+
+```
+{
+   "type": "record",
+   "name": "progress",
+   "namespace": "com.materialize.cdc",
+   "fields": [
+     {
+       "name": "lower",
+       "type": {
+         "type": "array",
+         "items": "long"
+       }
+     },
+     {
+       "name": "upper",
+       "type": {
+         "type": "array",
+         "items": "long"
+       }
+     },
+     {
+       "name": "counts",
+       "type": {
+         "type": "array",
+         "items": {
+           "type": "record",
+           "name": "counts",
+           "fields": [
+             {
+               "name": "time",
+               "type": "long"
+             },
+             {
+               "name": "count",
+               "type": "long"
+             }
+           ]
+         }
+       }
+     }
+   ]
+ }
+ ```
+
+### Example Materialize CDC schema
+
+You define the Materialize CDC format in the [Avro schema](../../sql/create-source/avro-kafka/#avro-format-details) when a source is created.
```

Comment:
The articles here are confusing. Is the "a" CDC format or "the" format? Should it be "your" format? Or "your instance of the Materialize CDC format"? I feel like probably *we* define "the" Materialize CDC format.

Reply:
Does "You specify the use of the Materialize CDC format in the Avro schema when a source is created" work?